changelog/pending/20230406--cli-plugin--add-plugin-run-command-for-running-plugin-binaries-directly.yaml

```diff
@@ -0,0 +1,4 @@
+changes:
+- type: feat
+  scope: cli/plugin
+  description: Add `plugin run` command for running plugin binaries directly.
```

Comment:
Maybe an "experimental" somewhere in there so that we can remove this if we change our minds?